.github/workflows/lint.yml

```
@@ -85,28 +85,24 @@ jobs:
         with:
           script: |
             return [...new Set(${{ steps.filter.outputs.modules_files }}
+              .filter(x => x.endsWith('main.nf'))
               .map(path => path
-                .replace('modules/nf-core/', '')
-                .split('/')
-                .slice(0, 2)
-                .filter(x => !x.startsWith('main.nf') && x !== 'tests' && x !== 'templates' && x !== 'meta.yml' && x !== 'environment.yml' && !x.endsWith('.md'))
-                .join('/'))
+                .replace('subworkflows/nf-core/', '')
```

Should still be modules, no?
```suggestion
                .replace('modules/nf-core/', '')
```

---

of course, thanks for catching that!